diff --git a/target/linux/mediatek/dts/mt7986a-xiaomi-redmi-router-ax6000-stock.dts b/target/linux/mediatek/dts/mt7986a-xiaomi-redmi-router-ax6000-stock.dts
index 114076dffc..197cb585cb 100644
--- a/target/linux/mediatek/dts/mt7986a-xiaomi-redmi-router-ax6000-stock.dts
+++ b/target/linux/mediatek/dts/mt7986a-xiaomi-redmi-router-ax6000-stock.dts
@@ -34,6 +34,6 @@
 
 	partition@2400000 {
 		label = "ubi";
-		reg = <0x2400000 0x5000000>;
+		reg = <0x2400000 0xF400000>;
 	};
 };
