diff --git a/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-ax3000t.dts b/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-ax3000t.dts
index d4b11cbe04..a0b3728d02 100644
--- a/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-ax3000t.dts
+++ b/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-ax3000t.dts
@@ -30,6 +30,6 @@
 	 */
 	partition@2800000 {
 		label = "ubi";
-		reg = <0x2800000 0x4e00000>;
+		reg = <0x2800000 0xce00000>;
 	};
 };
diff --git a/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-common.dtsi b/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-common.dtsi
index be600a1a79..e6647ff500 100644
--- a/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-common.dtsi
+++ b/target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-common.dtsi
@@ -386,9 +386,9 @@
 				read-only;
 			};
 
-			partition@7600000 {
+			partition@F600000 {
 				label = "KF";
-				reg = <0x7600000 0x40000>;
+				reg = <0xF600000 0x40000>;
 				read-only;
 			};
 		};
